src/algebra/symmetrized.lean

```diff
@@ -210,4 +210,47 @@ lemma mul_comm [has_mul α] [add_comm_semigroup α] [has_one α] [invertible (2
   a * b = b * a :=
 by rw [mul_def, mul_def, add_comm]
 
+--[has_add α] [has_mul α] [has_one α] [invertible (2 : α)]
+
+/- 2 commutes with every element of a ring -/
+lemma two_commute {α : Type*} [ring α] (a : α) : commute 2 a := begin
+  --convert commute.semiconj_by 2 a,
+  unfold _root_.commute,
+  rw [semiconj_by, mul_two, two_mul],
+end
```

Comment:
I think this can be proved with `(commute.one_left a).bit0`

Reply:
I get `invalid field notation, 'bit0' is not a valid "field" because environment does not contain 'semiconj_by.bit0'` - do I need to import something?

Reply:
Sorry, I meant `.bit0_left`

Reply:
Thanks.